core/backend.go

```diff
@@ -31,7 +33,7 @@ type MessageConstructor interface {
 // Verifier defines the verifier interface
 type Verifier interface {
 	// IsValidBlock checks if the proposed block is child of parent
-	IsValidBlock(block []byte) bool
+	IsValidBlock(rawProposal []byte) bool
```

Comment:
Does it make sense to rename every backend function which mentions `Block` into `Proposal` (also update comments accordingly)?

E.g.
```suggestion
	IsValidProposal(rawProposal []byte) bool
```

Reply:
Fixed 67e527c1b31f0819d976c8e0adb3eff491c6d7e9

Reply:
Fixed 67e527c1b31f0819d976c8e0adb3eff491c6d7e9